src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -46,7 +50,7 @@ public ResponseEntity fetchDisorder(@PathVariable("name") String name) {
      * @return a list of blogs given disorder name
      */
     @GetMapping("/disorders/{name}/blogs/")
-    public ResponseEntity fetchBlogs(@PathVariable("name") String name) {
+    public ResponseEntity fetchBlogsByDisorder(@PathVariable("name") String name) {
```

Comment:
Same as above, the endpoint could be slightly more clear as to show developers a more clear picture of the functionality.